qa/workunits/rbd/test_rbd_tasks.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash -ex
```

Comment:
@dillaman The preferred way is to use:
```
#!/usr/bin/env bash
set -ex
```
Because on some platforms (FreeBSD) bash is not installed to `/bin`.